README.md

```diff
@@ -4,15 +4,14 @@ Whole-body PBPK model of fluvoxamine as an inhibitor of CYP2C19 and CYP1A2
 This repository contains:
 
 - a PK-Sim snapshot (*.json) file of the current PBPK model
+- the PK-Sim project of the current PBPK model created with OSP version 10
+- Evaluation report created with OSP version 10
 - static content (e.g. text blocks, *.md files) as inputs for an evaluation plan
 - an evaluation plan (evaluation-plan.json) to create an evaluation report using the snapshot and static text blocks to display the performance of the model
 
-**The latest release of the snapshot of the model, the evaluation plan and the static content can be found [here](https://github.com/Open-Systems-Pharmacology/Fluvoxamine_CYP1A2_and_2C19_network-Model/releases/latest).**
+**The latest release of the snapshot of the model, the evaluation plan and the static content can be found  in this repository.**
```

Comment:
Why replacing the link to the latest release? The link was exactly where the latest release can be found :)

Reply:
Because it is an alternative version of the model.. maybe we should have a call about it.

Reply:
ok, with your explanation and having this model on a separate branch: should be OK I think